### PR TITLE
Strip orphan intro + disclaimer prose alongside stripped blockquotes

### PR DESCRIPTION
**Pull request by canfieldjuan**
Closes the **KNOWN GAP** that PR #625's \`test_regression_introductory_prose_orphaned_when_quote_stripped\` documented.

## The gap

#625's block-level stripper atomically removed contaminated blockquote blocks, but adjacent prose dangled:

\`\`\`
One reviewer on Reddit noted:

[block was here, now removed]

That quote is from a compensation discussion, not a CRM review.
\`\`\`

The intro (\`One reviewer on Reddit noted:\`) and the disclaimer (\`That quote is from a compensation discussion...\`) aren't blockquote-prefixed lines, so the prior stripper left them in place. They reference content that no longer exists — readers see broken prose.

## The fix

\`_remove_unmatched_quote_lines\` now widens each strip span to include:

1. **Orphan intro paragraph** — single-line paragraph ending with \`:\` immediately preceding the stripped block (with optional blank-line separator). Length-bounded (3-180 chars) so long substantive paragraphs that happen to end with \`:\` are preserved.
2. **Orphan disclaimer paragraph** — single-line paragraph immediately after the stripped block matching the acknowledged-misattribution patterns from the seo-geo-aeo-blog-post skill's v1.4.0 audit:
   - \`/that quote is from a .{0,60}? discussion, not a/i\`
   - \`/misattributed in the source data/i\`
   - \`/(quote\\|review\\|reviewer) (is\\|was) (from\\|on) a .{0,60}? (discussion\\|review\\|forum\\|context)/i\`
   - \`/(quote\\|review) is misattributed/i\`
3. **Blank separators** between the prose and the block.

Implementation uses a span-collection pass followed by a filter pass. Cleaner than single-pass-with-lookback and correctly handles edge cases where adjacent blocks could otherwise have overlapping spans.

## Tests

- \`test_full_block_stripped_when_pool_empty\` updated: removed count reflects intro+block (4 lines vs prior 2).
- \`test_regression_introductory_prose_orphaned_when_quote_stripped\` renamed and inverted (no longer documents a gap; asserts orphan-prose IS stripped).
- 4 new tests:
  - \`test_orphan_intro_alone_stripped_when_no_disclaimer\`
  - \`test_orphan_disclaimer_alone_stripped_when_no_intro\`
  - \`test_orphan_prose_preserved_for_kept_blocks\` (no cleanup when block kept)
  - \`test_orphan_intro_not_stripped_when_too_long\` (length heuristic)

**183 tests pass** across the three generator/backfill test files (was 179).

## Effect on existing posts

Generator change only. Existing published posts with orphan intro/disclaimer prose around old stripped quotes won't be auto-fixed by this PR. They'd be cleaned up if regenerated through the quality pipeline, or addressed by a one-off cleanup script.

## Test plan

- [x] 183 tests pass with \`ATLAS_SAAS_ENABLED=false ATLAS_DB_ENABLED=false\`
- [x] \`npm run build\` in atlas-churn-ui: clean
- [x] \`extracted_content_pipeline\` synced via \`extracted/_shared/scripts/sync_extracted.sh\`
- [x] Local pre-push review passes

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: atlas_brain/autonomous/tasks/b2b_blog_post_generation.py [hunk 1]
```diff
@@ -1571,94 +1640,99 @@ def _remove_unmatched_quote_lines(markdown: str, source_quotes: list[str]) -> tu
...
+                span_start = cursor
+
+        # Widen the span forward to swallow blank separators + an
+        # orphan disclaimer paragraph.
+        span_end = block_end
+        cursor = block_end
+        while cursor < n and not lines[cursor].strip():
+            cursor += 1
+        if cursor < n and _looks_like_orphan_disclaimer(lines[cursor]):
+            span_end = cursor + 1
```
[the remark this answers is not in the document]
Comment (canfieldjuan) [reply atlas_brain/autonomous/tasks/b2b_blog_post_generation.py:1735]: Addressed in 9127869c. `_looks_like_orphan_disclaimer` now refuses to classify lines starting with `>`, `#`, `-`, `*`, `+`, or `N.` ordered-list markers. Added `test_adjacent_blocks_disclaimer_shaped_second_block_not_absorbed` covering exactly the case you described (two adjacent blockquotes, second one's first line starts with disclaimer-shaped text) -- the second block is preserved instead of being absorbed into the first block's strip span. 184 tests pass.
